engine/src/main/java/org/entando/entando/web/user/model/UserRequest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ * 
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ * 
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.user.model;
+
+/**
+ *
+ * @author paddeo
+ */
+public class UserRequest {
```

Comment:
What does this get used for? Can we drop it?

Reply:
I created this object for mistake but, since I need it for users endpoints, I decided to keep it. But I'll drop it and re-create inside my "users endpoints" branch.